src/index.js

```diff
@@ -17,6 +17,7 @@ export class ESLintWebpackPlugin {
    * @param {Options} options
    */
   constructor(options = {}) {
+    this.key = Math.random().toString();
```

Comment:
Bad idea, you need set `key` in `apply`, use `compiler.name` as prefix, if it is `undefined` use `unknown`, if the next compiler also doesn't `name` use `unknown_1` and etc

Reply:
ok

Reply:
Updated @alexander-akait